src/appleseed-max-impl/appleseedrenderer/appleseedrenderer.cpp

```diff
@@ -1251,7 +1251,7 @@ ParamBlockDesc2 g_param_block_desc(
         p_accessor, &g_pblock_accessor,
     p_end,
 
-    ParamIdEnableImportanceSampling, L"enable_importance_sampling", TYPE_BOOL, P_TRANSIENT, 0,
+    ParamIdEnableLightImportanceSampling, L"enable_importance_sampling", TYPE_BOOL, P_TRANSIENT, 0,
```

Comment:
`L"enable_importance_sampling"` needs fixing too.

Reply:
changed it